prow_config.yaml

```diff
@@ -4,3 +4,20 @@ workflows:
   - app_dir: kubeflow/testing/workflows
     component: workflows
     name: unittests
+
+  - app_dir: kubeflow/kubeflow/testing/workflows
+    component: click_deploy_test
+    name: skipmeplease
+    job_types:
+    - periodic
+    include_dirs:
+    - bootstrap/*
+    - kubeflow/*
+    - testing/*
+    params:
+      workflowName: skipmeplease
+
+  - py_func: kubeflow.testing.ci.kf_unittests.create_workflow
```

Comment:
nit: Can you add a TODO: explaining what this function is doing? i.e. that it is a smoke test to verify that the workflow is actually triggered.

Reply:
Fixed.